.github/workflows/appetize.yml

```diff
@@ -0,0 +1,162 @@
+name: Appetize
+
+on:
+  workflow_dispatch:
+    inputs:
+      appetizeQueue:
+        description: Appetize queue to use (main/embed/all)
+        default: main
+        required: true
+      sdkPlatform:
+        description: Platform to use (android/ios/all)
+        default: all
+        required: true
+      sdkVersion:
+        description: SDK version to resolve Expo Go version (e.g. 42.0.0)
```

Comment:
Would be great if we could somehow fill this in by default. Or when omitted that it retrieves the `sdkVersion` automatically, for instance from the `runtime` or `snack-sdk`

Reply:
I think we can, but I'm not sure if that's a good idea. This workflow has the power of instantly breaking all appetizer emulators when using the wrong version. Forcing the initiator to think about the version it wants to use might be a good "sanity check" before deploying stuff. Not sure how you look at this though

Reply:
Okay fair enough. What will happen when you input an invalid version like "42.0.". Will this just fail with an understandable error?

Reply:
Never mind, I can see from the code that it checks the version. LGTM

Reply:
Yes, it will error that the SDK 42.0. can't be found 😬